src/stories/SegmentedPicker/type.ts

```diff
@@ -0,0 +1,5 @@
+export interface SegmentedPickerProps {
+  className?: string;
+  segments: string[];
```

Comment:
please change type from string to { label: string, value: T}, and use label to display, use value to mapping item

Reply:
change:add 
 interface Segment<T> {
  label: string;
  value: T;
}
and replace SegmentedPickerProps's segments